app/views/publishers/vacancies/edit_vacancy_sections/_edit_applying_for_the_job.html.slim

```diff
@@ -3,39 +3,22 @@
 = render ReviewComponent.new id: "applying_for_the_job" do |review|
   - review.heading title: t("publishers.vacancies.steps.applying_for_the_job"),
                    text: t("buttons.change"),
-                   href: organisation_job_build_path(vacancy.id, :applying_for_the_job)
+                   href: organisation_job_build_path(vacancy.id, :applying_for_the_job, back_to: local_assigns[:back_to])
     = review_section_tag(vacancy, %i[applying_for_the_job], [Publishers::JobListing::ApplyingForTheJobForm])
 
   - review.body do
-    = govuk_summary_list do |summary_list|
+    = validatable_summary_list(vacancy, show_errors: local_assigns[:show_errors], error_path: organisation_job_build_path(vacancy.id, :applying_for_the_job, back_to: local_assigns[:back_to])) do |list|
       - unless current_organisation.group_type == "local_authority"
-        - summary_list.row do |row|
-          - row.key text: t("jobs.enable_job_applications")
-          - row.value text: vacancy.enable_job_applications? ? "Yes" : "No"
+        - list.row :enable_job_applications
 
       - if vacancy.enable_job_applications?
-        - summary_list.row do |row|
-          - row.key text: t("jobs.personal_statement_guidance")
-          - row.value text: vacancy.personal_statement_guidance.presence || t("jobs.not_defined")
-
+        - list.row :personal_statement_guidance, optional: true
       - else
-        - summary_list.row do |row|
-          - row.key text: t("jobs.application_link")
-          - row.value text: vacancy.application_link.present? ? open_in_new_tab_link_to(vacancy.application_link, vacancy.application_link, "aria-label": t("jobs.aria_labels.apply_link")) : t("jobs.not_defined")
+        - list.row :application_link, if_present: open_in_new_tab_link_to(vacancy.application_link, vacancy.application_link, "aria-label": t("jobs.aria_labels.apply_link"))
```

Comment:
Trying to understand what `if_present` is up to.

The RowComponent will do its default logic for displaying the attribute, unless there is an `if_present` present, in which case it will display it according to that (as long as the attribute is present). Right?

Reply:
Yeah there's some logic in some of the templates where "if the attribute is set, display this other thing".  It's not very elegant and I'm open to other approached.

Reply:
A low-hanging fruit improvement that would make it easier for me to read and understand in the view (without having to go read the component), is to call it `if_attribute_present`.

Reply:
How about `value_if_attribute_present`?